File: StressTracker5001App/src/dto/comment.dto.ts
import type { UserDto } from "./user.dto";

export interface CommentDto {
  id: number;
  content: string;
  userId: number;
  user: UserDto;
  createdAt: string;
  updatedAt: string;
}

export interface CommentCreateDto {
  content: string;
}

export interface CommentUpdateDto {
  content: string;
}

File: StressTracker5001Server/DTOs/Comment/CommentDto.cs
using StressTracker5001Server.DTOs.User;

namespace StressTracker5001Server.DTOs.Comment
{
    public class CommentDto
    {
        public required int Id { get; set; }
        public required string Content { get; set; }
        public required int UserId { get; set; }
        public required UserDto User { get; set; }
        public required DateTime CreatedAt { get; set; }
        public required DateTime UpdatedAt { get; set; }
    }
}
